ignition-overrides/complete-installation.sh.template

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This script is called by the complete-installation.service
+# It will complete the installation by executing coreos-installer and reboot the node
+# Make sure to
+
+echo "Waiting for /opt/openshift/master.ign"
```

Comment:
@eranco74 I think u better wait either to:
https://github.com/openshift/installer/pull/4482/files#diff-0c7e4b25006007118d630547ef6b3502118f0561220a63b6f9caf910dc68980bR52

or till bootkube.sh finishes.
Just not to have a race where master.ign is partially written since u dont write to tmp file and then move - so it's not atomic.